README.md

```diff
@@ -1,86 +1,51 @@
-The `notebooks` folder contains jupyter notebooks for testing and demonstrating functions of the `scIB` package as well as notebooks
-for preprocessing of the data.
-
-
-
-## Installation
-To reproduce the results from this study, three different conda environments are needed.
-There are different environments for the python integration methods, the R integration methods and
-the conversion of R data types to anndata objects.
-
-For the installation of conda, follow [these](https://conda.io/projects/conda/en/latest/user-guide/install/index.html) instructions
-or use your system's package manager. The environments have only been tested on linux operating systems
-although it should be possible to run the pipeline using Mac OS.
-
-To create the conda environments use the `.yml` files in the `envs` directory.
-To install the envs, use
-```bash
-conda env create -f FILENAME.yml
-``` 
-To set the correct paths so that R the correct R libraries can be found, copy `env_vars_activate.sh` to `etc/conda/activate.d/`
-and `env_vars_deactivate.sh` to `etc/conda/deactivate.d/` to every environment.
-As `Saucie` is not available as a python package for the scIB-python env, it needs to be installed manually from [here](https://github.com/KrishnaswamyLab/SAUCIE).
-Furthermore, you need to install `kBET` in the `scIB-python` env with `r-devtools` as described [here](https://github.com/theislab/kBET).
-
-In the `scIB-R-integration` environment, R packages need to be installed manually.
-Activate the environment and install the packages `scran`, `Seurat` and `Conos` in R. `Conos` needs to be installed using R devtools.
-See [here](https://github.com/hms-dbmi/conos).
-We used these versions of the R integration methods in our study:
 ```
-harmony_1.0
-Seurat_3.2.0
-conos_1.3.0
-liger_0.5.0
-batchelor_1.4.0
+pip install .
 ```
 
-## Running the integration methods
-This package allows to run a multitude of single cell data integration methods in both `R` and `python`.
-We use [Snakemake](https://snakemake.readthedocs.io/en/stable/) to run the pipeline.
-The parameters of the run are configured using the `config.yaml` file.
-See the `DATA_SCENARIOS` section to change the data used for integration.
-The script expects one `.h5ad` file containing all batches per data scenario.
+Additionally, in order to run the R package `kBET`, you need to install it through R.
+
+```R
+devtools::install_github('theislab/kBET')
+```
 
-To load the config file run `snakemake --configfile config.yaml`.
-Define the number of CPU threads you want to use with `snakemake --cores N_CORES`. To produce an overview of tasks that will be run, use `snakemake -n`.
-To run the pipeline, simply run `snakemake`.
+We recommend to use a conda environment or something similar, so that python and R dependencies are in one place.
+Please also check out [scIB pipeline](https://github.com/theislab/scib-pipeline.git) for ready-to-use environments.
```

Comment:
as the python integration methods are included in this package, we should include some statement about installation of additional packages being required to use several functions in the `scIB.ig` submodule. Did we add a particular install option that also installs the required packages for this or not?

Reply:
We don't really have a solution out-of-the-box. And that is a bit more complicated as well, as we'd need to specify which Integration package should be installed. It would be easier for the user to install their method of choice.

Reply:
yeah, can't just install all as versions clash... true. Maybe if we add a quick statement about that it'll be fine.